internal/rest/avi_obj_httpps.go

```diff
@@ -188,7 +188,7 @@ func (rest *RestOperations) AviHttpPSBuild(hps_meta *nodes.AviHttpPolicySetNode,
 		protocol := "HTTPS"
 		redirect_action.StatusCode = &hppmap.StatusCode
 		redirect_action.Protocol = &protocol
-		redirect_action.Port = &hppmap.RedirectPort
+		redirect_action.Port = uint32(hppmap.RedirectPort)
```

Comment:
@DixitAakash This field is changed from *int32 to uint32. Did we test this AKO image with older Avi Controller where these SDK changes are not there?

Reply:
@akshayhavile I have checked with controller version 22.1.3 and the changes are working fine without any issues.